docs/pages/jupedsim/jupedsim_install_on_windows.md

```diff
@@ -110,24 +27,31 @@ cd vcpkg
 .\bootstrap-vcpkg.bat
 ```
 
-For `JuPedSim` following C++ libraries are needed:
+For `jpscore` following C++ libraries are needed:
```

Comment:
Is `jpscore` only `jpscore` or the repo with `jpsreport`?

Reply:
added new commit for this